app/js/editQuestion.js

```diff
@@ -0,0 +1,108 @@
+let authorised = false;
+let user = getCookie('userEmail');
+
+getUser(user).then((user) => {
+    if (user.data.isAdmin === '1') {
+        authorised = true;
+    }
+});
+
+// populate dropdown menu with available tests
+populateHandlebars('#test_id', 'js/templates/testDropdown.hbs', 'test');
+
+/**
+ * Edit a question in the database
+ * @param {string} id The question you want to edit
+ * @param {object} newQuestionData The data you are changing
+ * @returns {Promise<void>}
+ */
+async function editQuestion(id, newQuestionData) {
+    return await sendData(jsonToFormData(newQuestionData), `/question/${id}/edit`);
+}
+
+/**
+ * Get a question from the database
+ * @param {string} id The question you want to get
+ * @returns {Promise<object>} the question data
+ **/
+async function getQuestionById(id) {
+    const { data } = await getData("/question");
+    return data.filter(question => question.id === String(id));
+}
+
+/**
+ * Get the answer of a specific question
+ * @param {string} id The question you would like the answer of
+ * @returns {Promise<string>} the answer number
+ */
+async function getAnswerByQuestionId(id) {
+    const { data } = await getData(`/answer/${id}`);
+    return data.answer;
+}
+
+// We are listening to the load event so that we can use async/await.
```

Comment:
you can use async without an event listener, you can just create an async named function then call it straight away